packages/cli/src/utils/compiler.ts

```diff
@@ -77,10 +79,54 @@ export const compilePlatformApp = async ({ outputName, projectPath = '' }: Compi
         },
     };
 
-    return build({
+    const jsHashPlugin: PluginOption = {
+        name: 'js-hash',
+        enforce: 'post',
+        generateBundle(_options, bundle) {
+            bundle['index.js'].fileName = 'settings.js';
```

Comment:
```suggestion
            bundle['index.js'].fileName = 'index.js';
```
The entry is `index` not `settings` or am i wrong?